src/app/technical-record/vehicle-summary/vehicle-summary.component.html

```diff
@@ -48,7 +38,7 @@
     </dd>
     <dd class="govuk-summary-list__value"></dd>
   </div>
-  <div class="govuk-summary-list__row">
+  <div class="govuk-summary-list__row" *ngIf="isStandardVehicle(activeRecord.vehicleType)">
```

Comment:
Can we create a container component for it please (for HGV, TRL, PSV) as discussed?
It will make the template lighter and easy to scan, we can name it vtm-vehicle-summary-regular
regular is what is used by the business when they talk about hgv,psv & trl regarding specialist test.

Reply:
@JonLazariniDD @alleypa we need to sort this out please. jon you can check history about what i did and what i changed

Reply:
let's have a chat about this on Monday, i understood what we did as mentioned in the general but unfortunately it doesn't make sense as we are duplicating effort once again if we had update that. We are rewriting the same markup and it doesn't make sense because we could only write it once in fact. I don't know why this has been changed.
We can leave it like that if the forms are rendered properly (for all the specialist tests).
We have a ticket in the backlog if we want to refactor it.
Now since test result come into picture we can see that some of these components are also used in test result so it makes sense to start refactoring as we develop.
We can leave it for now but we will still have to check whether we have the required id for automation test, move the isStandardVehicle into the service (with appropriate unit test) and see check that we shouldn't render notes section entirely based on the vehicle types mentioned otherwise we will introduce a bug and I believe that section shouldn't be shown as we had a similar bug fixed by Umer.